upgrading/rosa-upgrading-sts.adoc

```diff
@@ -0,0 +1,31 @@
+[id="rosa-upgrading-sts"]
+= Upgrading ROSA clusters with STS
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-upgrading-sts
+
+toc::[]
+
+[id="rosa-lifecycle-policy_{context}"]
+== Life cycle policies and planning
+
+To plan an upgrade, review the xref:../rosa_policy/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
+
+include::modules/rosa-upgrading-4_7_to_4_8_prerequisites.adoc[leveloffset=+1]
+
+[id="rosa-sts-upgrading-a-cluster-with-sts"]
+== Upgrading a ROSA cluster that uses STS
+
+There are three methods to upgrade {product-title} (ROSA) clusters that uses the AWS Security Token Service (STS):
+
+* Manual upgrades through the `rosa` CLI
+* Manual upgrades through the {cloud-redhat-com} console
+* Automatic upgrades through the {cloud-redhat-com} console
+
+[NOTE]
+====
```

Comment:
Nice job of anticipating that some customers want information about the alternative.